src/styles/main.scss

```diff
@@ -1,3 +1,54 @@
-body {
-  margin: 0;
+$days-of-week: mon, tue, wed, thu, fri, sat, sun;
+$start-day: fri;
+$start-day-index: index($days-of-week, $start-day);
+$cell-border-width: 1px;
+$cell-size: 100px - ($cell-border-width * 2);
+$cell-font-size: 30px;
+$cell-padding: 10px;
+$month-length: 31;
+
+$calendar-width: 7 * $cell-size + (6 * $cell-border-width + 2 * $cell-padding);
+
+.calendar {
+  max-width: $calendar-width;
+  margin: 0 auto;
+  padding: $cell-padding;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: $cell-border-width;
+
+  &__day {
+    width: $cell-size;
+    height: $cell-size;
+    border: $cell-border-width solid #000;
+    background-color: #eee;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    &:hover {
+      cursor: pointer;
+      background-color: pink;
+      transition: transform 0.5s;
```

Comment:
you should add `transition` for element, not inside `:hover` to make it work in both ways